schema/artist/index.js

```diff
@@ -1,5 +1,7 @@
 import _ from 'lodash';
 import Image from '../image';
+import ArtworkSorts from '../sorts/artwork_sorts'
```

Comment:
Let me know what you think. We'll probably have a bunch of schemas that include artworks, we probably want to be able to sort those occurrences by some common artwork sort types.